dash/dash.py

```diff
@@ -678,7 +678,9 @@ def serve_component_suites(self, package_name, path_in_package_dist):
         })[path_in_package_dist.split('.')[-1]]
 
         loader = pkgutil.get_loader(package_name)
-        self.logger.info(
+        msg = getattr(
+            self.logger, 'info' if self.config.serve_locally else 'debug')
```

Comment:
do we actually get here if we're not serving locally?

Reply:
Interesting question. the flag that I really want is the `debug` argument but in this function `self.config` is easier to access. 

the logic behind using serve_locally is from here https://github.com/plotly/dash/blob/a42b712e1a88d17400726b5b28ef20a36e826d21/dash/dash.py#L1557-L1560

it's False by default and set to True when debug is True

Reply:
I tested with debug=True/False,  the logging level get changed as expected